content/api/api-packages.md

```diff
@@ -8,199 +8,427 @@ alwaysopen: false
+{{< button theme="info" href="https://app.metricly.com/swagger-ui.html#!/packages/getUsingGET_2" >}} GET {{< /button >}} Use this endpoint to get an installed package.
+
+{{% expand "View method details."%}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|-------------|----------------|-----------|----------------------|
+| id | path | string | Unique ID of the package |
 
-### POST to /packages/install
-Install packages via zip file (Content-Type: multipart/form-data).
+### Request URL
+
+ `https://app.metricly.com/packages/{id}`
+
+### CURL
+
+The following example only requires the **id** (_not_ **packageId**) to be added to the Request URL.
 
 ```
-curl -X POST -u username:password https://api.us.cloudwisdom.virtana.com/packages/install -H 'content-type: multipart/form-data' -F file=@package.zip
+curl -X GET --header 'Accept: application/json' 'https://app.metricly.com/packages/9b20b44c-aa37-39fa-b46b-9c17237337fe'
+
 ```
 
-## DELETE
+### Response Body
 
-### DELETE from /packages/{id}
-Deletes specified package.
+The following response body
 
 ```
-curl -X DELETE --header 'Accept: */*' 'https://us.cloudwisdom.virtana.com/packages/123packageid123'
+{
+  "package": {
+    "id": "9b20b44c-aa37-39fa-b46b-9c17237337fe",
+    "tenantId": "0fa5384a-3b36-4766-ad68-43e7c6af54f6",
+    "packageId": "netuitive.packages.aws.cost",
+    "version": "0.2.0",
+    "name": "Netuitive Packages Total Cost",
+    "description": "A set of assets to showcase total cost data captured by Metricly",
+    "authorName": "Netuitive Research",
+    "authorEmail": "research@netuitive.com",
+    "logo": "http://assets.app.netuitive.com/pkg/logo/netuitive/banner_logo.png",
+    "downloadUrl": "https://github.com/netuitive-community-packages/netuitive-packages-aws-cost/archive/master.zip",
+    "userEmail": null,
+    "created": "2018-07-27T01:26:00Z",
+    "updated": "2018-07-27T01:26:00Z",
+    "tags": [],
+    "policies": [],
+    "analyticConfigurations": [],
+    "dashboards": [
+      {
+        "id": "54fc1a1b-d762-32b7-af4b-d6cd4e2dabd5",
+        "name": null,
+        "description": null,
+        "layout": null,
+        "properties": null,
+        "widgets": null,
+        "type": null,
+        "private": true
+      }
+    ]
+  }
+}
 ```
+{{% /expand %}}
+
+---
+
+## Modify an Existing Community Package
+
+CloudWisdom has a [community GitHub for packages](https://github.com/netuitive-community-packages) available for download. You can install these via the API,  customize them to your liking, or even contribute with a completely new package. Let's simply modify an existing package to fit our needs.
+
+{{% expand "View method details."%}}
+
+**Prerequisite**: If you haven't already, install the [CLI](https://docs.metricly.com/api/api-cli/).
+
+1\. Go to the [community GitHub for packages](https://github.com/netuitive-community-packages) and download the **netuitive-packages-linux** package as a .zip file.
```

Comment:
We should encourage them to clone it using the git command in hopes they will track their changes. Downloading as a zip can be optional.
```suggestion
1\. Go to the [community GitHub for packages](https://github.com/netuitive-community-packages) and clone the **netuitive-packages-linux** package to your local directory i.e. ``git clone https://github.com/netuitive-community-packages/netuitive-packages-linux.git``. Optionally, download the package as a zip file and unzip it.
```